dev/com.ibm.ws.messaging.open_jms20_fat/fat/src/com/ibm/ws/messaging/JMS20/fat/FATSuite.java

```diff
@@ -36,8 +40,8 @@
 
         DummyTest.class,
         LiteBucketSet1Test.class,
-        LiteBucketSet2Test.class,
-        JMSMBeanTest.class,
+        LiteBucketSet2Test.class, // 2 tests disabled for jakarta
+        JMSMBeanTest.class, // not run in jakarta: j2ee-management is not supported
```

Comment:
We need to consult with messaging team to determine whether the messaging runtime is sufficiently verified with these tests disabled, and if not, how to verify the same function without j2ee-management. Also, do we need these comments?  The annotations speak for themselves.  I'm thinking to remove all of them.